CryptoPkg/Library/Include/CrtLibSupport.h

```diff
@@ -437,6 +472,66 @@ strcat (
   const char  *strSource
   );
 
+int
+fflush (
+  FILE  *stream
+  );
+
+int
+ferror (
+  FILE  *fp
+  );
+
+int
+fseek (
+  FILE  *stream,
+  long  offset,
+  int   whence
+  );
+
+int
+feof (
+  FILE  *fp
+  );
+
+char *
+fgets (
+  char  *str,
+  int   n,
+  FILE  *stream
+  );
+
+int
+ftell (
+  FILE  *fp
+  );
+
+int
+fputs (
+  const char  *str,
+  FILE        *stream
+  );
+
+char *
+strdup (
+  char  *strSource
+  );
+
+void *
+bsearch (
+  const void *key,
+  const void *base,
+  size_t nitems,
+  size_t width,
+  int         ( *cmp )(const void *, const void *)

```

Comment:
```suggestion
  size_t nmemb,
  size_t size,
  int       ( *compar )(const void *, const void *)
```

Use the same names as bsearch(3).